### ISSUE TITLE
Skip resource compression during test build compilation

### ISSUE THREAD

**Issue by sadko4u** [lsp-plugins/lsp-plugins#366]

**Comment by sadko4u** [comment]
Now it's possible to build test binaries without generating and archiving builtin resources.

This can be done by the following commands:
```
make config ... FEATURES= TEST=1
make
```

After that, if we launch some plugin, we get an error of missing manifest file which is a part of builtin resources:
```
.build/host/lsp-plugin-fw/lsp-plugins-test mtest standalone --args comp_delay_stereo

--------------------------------------------------------------------------------
Launching manual test 'standalone'
--------------------------------------------------------------------------------
[ERR] No manifest.json found in resources
Manual test '.standalone' assertion has failed at file test/mtest/standalone.cpp, line 207:
  result == 0
Manual test 'standalone' has failed, execution time: 0.00 s

```

But additional command line switch `--external-resources` fixes the problem:
```
.build/host/lsp-plugin-fw/lsp-plugins-test mtest standalone --args comp_delay_stereo --external-resources
```

**Comment by PROPHESSOR** [comment]
I added `launch_plugin.sh` file to `lsp-plugins::lsp-plugins#feat/prophessor` to simplify this action. It may be useful for someone

**Comment by sadko4u** [comment]
Available in 1.2.15
